Support of jberet-core 1.2.0-Final

bflorat opened the pull request:
Support of last final release of jberet-core under EAP6. 

Arquillian tests passed.

Commit:
Support of jberet-core 1.2.0-Final

finsterwalder commented:
Please review this pull-request, since it would be nice to upgrade to the current version.

finsterwalder commented:
fcorneli told be, that he no longer maintains this project, so the pull request will not be merged.
Switch to bflorat's project instead.

Commit:
Batch-jberet subsystem start up before datasource lead to application start failed
Solution:
1. Provide a way to find datasource service instead of lookup it by jndi name.
2. Add JobRepositoryService to provide job repository, also add datasource dependency on JobRepositoryService to avoid JobRepositoryService start up before datasource.

Commit:
Add bug fix description in 1.0.3

YinghuiCui committed:
Update README.md

bflorat committed:
Merge pull request #1 from YinghuiCui/master

[Bug fix] Batch-jberet subsystem starting up before datasource ready leads to application start failed
